app/views/downloads/_download_preface.html.erb

```diff
@@ -1,66 +1,62 @@
-#{link_to(s.series.title, series_url(s.series))}"}.join(", ").html_safe %></dd>
+        <dt><%= t(".series") %></dt>
+        <dd><%= safe_join(series_list.map { |s| t(".series_list_html", position: s.position, series_link: link_to(s.series.title, series_url(s.series))) }, t("support.array.words_connector")) %></dd>
       <% end %>
       <% unless @work.approved_collections.empty? %>
-        <dt><%= ts("Collections") %>:</dt>
-        <dd><%= @work.approved_collections.map {|c| link_to c.title, collection_url(c)}.join(",
-").html_safe %></dd>
+        <dt><%= t(".collections") %></dt>
+        <dd><%= safe_join(@work.approved_collections.map { |c| link_to(c.title, collection_url(c)) }, t("support.array.words_connector")) %></dd>
       <% end %>
-      <dt><%= ts("Stats") %>:</dt>
+      <dt><%= t(".stats") %></dt>
       <dd>
-        <%= ts("Published") %>: <%= l(@work.first_chapter.published_at) %>
+        <%= t(".published", date: l(@work.first_chapter.published_at)) %>
         <% if @work.first_chapter.published_at < @work.revised_at.to_date %>
-          <%= @work.is_wip ? ts("Updated") : ts("Completed") %>: <%= l(@work.revised_at.to_date) %>
+          <%= @work.is_wip ? t(".updated", date: l(@work.revised_at.to_date)) : t(".completed", date: l(@work.revised_at.to_date)) %>
         <% end %>
-        <%= ts("Words") %>: <%= number_with_delimiter(@work.word_count) %>
-        <%= ts("Chapters") %>: <%= chapter_total_display(@work) %>
+        <%= t(".words", count_with_delimiters: number_with_delimiter(@work.word_count)) %>
+        <%= t(".chapters", chapter_total_display: chapter_total_display(@work)) %>
       </dd>
     </dl>
     <h1><%= @work.title %></h1>
-    <div class="byline"><%= ts("by") %> <%= byline(@work, visibility: 'public', only_path: false) %></div>
+    <div class="byline"><%= t(".byline_html", byline_names_link: byline(@work, visibility: "public", only_path: false)) %></div>
     <% unless @work.summary.blank? %>
-      <p><%= ts("Summary") %></p>
-      <blockquote class="userstuff"><%=raw sanitize_field(@work, :summary) %></blockquote>
+      <p><%= t(".summary") %></p>
+      <blockquote class="userstuff"><%= raw sanitize_field(@work, :summary) %></blockquote>
     <% end %>
 
     <% unless @work.notes.blank? && @work.endnotes.blank? %>
-      <p><%= ts("Notes") %></p>
+      <p><%= t(".notes") %></p>
       <% unless @work.notes.blank? %>
-        <blockquote class="userstuff"><%=raw sanitize_field(@work, :notes) %></blockquote>
+        <blockquote class="userstuff"><%= raw sanitize_field(@work, :notes) %></blockquote>
       <% end %>
       <% unless @work.endnotes.blank? %>
         <div class="endnote-link">
-          <%= ts("See the end of the work for") %> <% unless @work.notes.blank? %><%= ts("more") %><% end
-%> <%= link_to ts("notes"), "#endnotes" %>
+          <% t(".see_end_notes_html", end_notes_link: link_to(@work.notes.present? ? t(".more_notes_link") : t(".notes_link"), "#endnotes")) %>
```

Comment:
This is missing an `=` sign to display the text, i.e. `<%= t(".see_end_notes_html"`. The end notes for `download_chapter` also has the [same issue](https://github.com/otwcode/otwarchive/pull/5198/files#diff-70e04a9524cc7a5f2cd08154551ba236a97ddcf1d1e4de81e15fe013ea477c61R22).